webpack.config.browser.js

```diff
@@ -35,11 +35,11 @@ const config = {
     rules: [
       {
         test: /\.m?(ts|js)$/,
-        exclude: /node_modules/,
+        exclude: /node_modules\/(?!(stellar-base|js-xdr))/,
```

Comment:
is this related to soroban-client exporting these(base and xdr)?

Reply:
correct, and it needs to transpile them because they use newer JS features